audit/README.md

```diff
@@ -1,9 +1,9 @@
 # cool-accounts - audit subdirectory #
 
 This subdirectory contains Terraform code to provision the COOL
-"audit" account.  It creates an IAM role that allows sufficient
+"Audit" account.  It creates an IAM role that allows sufficient
```

Comment:
We don't put the account names in quotes elsewhere, so I suggest removing the quotes here for consistency:
```suggestion
Audit account.  It creates an IAM role that allows sufficient
```

Reply:
Weird- this one didn't show up in my files view, so I couldn't include it in the batch commit (https://github.com/cisagov/cool-accounts/pull/128/commits/a0b88bc753ebbe8809fef62c457003261359a009), which is why I added my own identical suggestion.  I'll close this conversation out since it is covered by https://github.com/cisagov/cool-accounts/pull/128/commits/a0b88bc753ebbe8809fef62c457003261359a009.